asmcomp/arm/arch.ml

```diff
@@ -53,8 +54,10 @@ let (arch, fpu, thumb) =
     | EABI, "armv6"    -> ARMv6,   Soft,      false
     | EABI, "armv6t2"  -> ARMv6T2, Soft,      false
     | EABI, "armv7"    -> ARMv7,   Soft,      false
+    | EABI, "armv8"    -> ARMv8,   Soft,      false
     | EABI, _          -> ARMv4,   Soft,      false
     | EABI_HF, "armv6" -> ARMv6,   VFPv2,     false
+    | EABI_HF, "armv8" -> ARMv8,   VFPv3,     true
```

Comment:
As a note, I checked about VFPv3-D16 on ARMv8: it turns out such a thing doesn't exist.  FP implementations on v8 must have 32 registers.
I wonder why our ARMv7 default is for VFPv3-D16.